worlds/civ_6/Rules.py

```diff
@@ -0,0 +1,53 @@
+from typing import TYPE_CHECKING, Callable, List, Dict
+from BaseClasses import CollectionState
+from .Items import get_item_by_civ_name
+from .Data import get_boosts_data
+from .Enum import CivVICheckType
+from .ProgressiveDistricts import convert_items_to_have_progression
+
+from worlds.generic.Rules import forbid_item, set_rule
+
+
+if TYPE_CHECKING:
+    from . import CivVIWorld
+
+
+def create_boost_rules(world: 'CivVIWorld'):
+    boost_data_list = get_boosts_data()
+    boost_locations = [location for location in world.location_table.values() if location.location_type == CivVICheckType.BOOST]
+    for location in boost_locations:
+        boost_data = next((boost for boost in boost_data_list if boost.Type == location.name), None)
+        world_location = world.multiworld.get_location(location.name, world.player)
+        forbid_item(world_location, "Progressive Era", world.player)
+        if not boost_data or boost_data.PrereqRequiredCount == 0:
+            continue
+
+        set_rule(world_location, lambda state, prereqs=boost_data.Prereq, required_count=boost_data.PrereqRequiredCount: has_required_items(state, prereqs, required_count, world.player))
+
+
+def has_required_items(state: CollectionState, prereqs: List[str], required_count: int, player: int) -> bool:
+    world: 'CivVIWorld' = state.multiworld.worlds[player]
```

Comment:
I have been told in the past that this pattern (getting the world class from state) isn't preferred

Reply:
Ah nice, I'll pass it in instead 👍